saleor/graphql/invoice/mutations.py

```diff
@@ -19,7 +19,7 @@ class InvoiceRequest(ModelMutation):
     class Meta:
         description = "Request an invoice for the order using plugin."
         model = models.Invoice
-        permissions = (OrderPermissions.MANAGE_ORDERS,)
+        # permissions = (OrderPermissions.MANAGE_ORDERS,)
```

Comment:
What about this line?

Reply:
reverted it very quickly after push but you were fast enough to spot it :police_officer: